Core.ApplicationServices/System/Write/ItSystemWriteService.cs

```diff
@@ -279,5 +298,10 @@ private Result<ItSystem, OperationError> GetSystemAndAuthorizeAccess(Guid system
                 .GetSystem(systemUuid)
                 .Bind(WithWriteAccess);
         }
+
+        private Result<ItSystem, OperationError> WithDBSWriteAccess(ItSystem system)
+        {
+            return WithWriteAccess(system); //Placeholder for now
```

Comment:
If the task is ready, and Mikael still hasn't responded let's create a task for the access control and paste the link here